DOCS-425 update Data-in-memory.md (#88)
aerospike/kubernetes-operator: docs/Data-in-memory.md

description: Data In Memory
---

> A data in memory (DIM) configuration is useful in many situations. This is also an easy way to test an Aerospike cluster without having to configure separate storage. For more information on data-in-memory and other storage configurations, [see the Aerospike documentation for namespace storage configuration](https://docs.aerospike.com/docs/operations/configure/namespace/storage/index.html).

Reviewer: ```suggestion
A data in memory (DIM) configuration is useful in many situations. For instance, it is an easy way to test an Aerospike cluster without having to configure separate storage. For more information on data-in-memory and other storage configurations, [see the Aerospike documentation for namespace storage configuration](https://docs.aerospike.com/docs/operations/configure/namespace/storage/index.html).
```
